pkg/branding/brandedlogo.go

```diff
@@ -0,0 +1,38 @@
+package branding
+
+import (
+	"embed"
+	"encoding/base64"
+
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+const (
+	rhacslogoFile    = "files/red-hat-acs-logo-rgb.png"
+	stackroxLogoFile = "files/StackRox_Logo_Wide_DkBlue.png"
```

Comment:
Nit: Should we take this opportunity to adhere to the file naming convention?
AFAIK, most of the repo is using lower_case_underscore_spacers, correct? 
We could rename the ACS logo to use underscores and rename the StackRox logo to be all lower case.

Reply:
I'm okay with that. The current filenames are how I found the files originally and don't need to be preserved.

Reply:
+1